doc/developer_guide.rst

```diff
@@ -47,6 +47,13 @@ Integration Tests
 
    As a Docker container with a test databases needs to be started for the integration tests, it may take a bit before the tests themselves start executing. After the tests have been run, the database will be shut down again.
 
+   If you are running Docker inside of a VM and are running the integration tests with `ssl.CERT_REQUIRED`, inside your VM map the `exasol-test-database` to the associated IP address.
+   This mapping is required due to how the certificate was created & for hostname resolution.
+
+    .. code-block:: shell
+
+        echo "127.0.0.1 exasol-test-database" | sudo tee -a /etc/hosts
+
```

Comment:
Another solution would be to let the integration tests spawn a new docker container which prepares this environment.....

Reply:
but then we would need to run the tests in this docker container

Reply:
I think, for the moment, we stay with the current variant, but we should have a ticket for a better solution, maybe the solution is also in itde, that we make it configurable what name is in the certificate @ArBridgeman please create a ticket that we think about how to improve this

Reply:
Feel free to edit/comment https://github.com/exasol/integration-test-docker-environment/issues/462